src/data/zwave_js.ts

```diff
@@ -58,6 +86,38 @@ export const fetchNodeStatus = (
     node_id,
   });
 
+export const fetchNodeConfigParameters = (
+  hass: HomeAssistant,
+  entry_id: string,
+  node_id: number
+): Promise<ZWaveJSNodeConfigParams[]> =>
+  hass.callWS({
+    type: "zwave_js/get_config_parameters",
+    entry_id,
+    node_id,
+  });
+
+export const setNodeConfigParameter = (
+  hass: HomeAssistant,
+  entry_id: string,
+  node_id: number,
+  property: number,
+  value: number,
+  property_key: number | null
```

Comment:
Change this to 

```suggestion
  property_key?: number
```

undefined keys are dropped when serializing an object to JSON.